dotnetv3/Dockerfile

```diff
@@ -2,7 +2,14 @@
 # Status: Beta
 # GA updates: https://github.com/awsdocs/aws-doc-sdk-examples/issues/4126 
 FROM mcr.microsoft.com/dotnet/sdk:6.0
-COPY . /dotnetv3/
-RUN cd /dotnetv3/SES && \
-    dotnet build
+
+#uncomment these lines to also enable the .NET 5.0 SDK if running 5.0 examples:
```

Comment:
It would be nice if Docker had conditional COPY based on an arg.

Reply:
A good point! I can try something fancier for the next version. I hope that at some point soon we can drop the 5.0 altogether, though.